relay/pkg/chain/chain.go

```diff
@@ -51,4 +51,14 @@ type Relay interface {
 		newBestHeader []byte,
 		limit *big.Int,
 	) error
+
+	// MarkNewHeaviestPreflight performs a preflight call of the
+	// MarkNewHeaviest method to check whether its execution will
+	// succeed.
```

Comment:
Worth adding information on what result is expected from a successful and failed preflight call.

Reply:
https://github.com/keep-network/tbtc/pull/771/commits/e4f08c6cbbf84b67056a0af6792788de659c8fcc